File: Sum of Digit is Pallindrome or not/sum of digit in pallindrome or not.py
def isDigitSumPalindrome(N):
    f = str(N)
    s=0
    for i in f:
        s+=int(i)
        
    d = str(s)

    j=d[::-1]
    if j == d:
        return 1
    else : 
        return 0

n= 1234

print (isDigitSumPalindrome(n))

File: Sum of Digit is Pallindrome or not/sum of digit in pallindrome or not.cpp
#include <iostream>
#include <string>
#include <algorithm>
using namespace std;
class Solution {
  public:
    int isDigitSumPalindrome(int N) {
        string f =to_string(N);
        int sum=0;
        
        for (char i : f){
            sum += i - '0';
        }
        string g = to_string(sum);
        string h = g;
        reverse(h.begin(),h.end());
        if (g == h){
            return 1;
        }
        else{
            return 0;
        }       
    }
};
int main() {
    int n = 61245;
    Solution p;
    cout<<p.isDigitSumPalindrome(n);
}